### DISCUSSION TITLE
How use veevalidate with inertiajs

### DISCUSSION THREAD

**Discussion by majweb** [inertiajs/inertia#1587] [Help (Vue)]
Can I use veevalidate with sens form. How?

**Comment by 300code** [comment]
Sure you can, you have to use proxy for props as middleware

**Comment by majweb** [comment]
How?

pt., 7 mar 2025, 04:02 użytkownik Mido Hamed ***@***.***>
napisał:

**Comment by 300code** [comment]
Use intus or yup as schema--Sent from Yandex Mail for mobile07.03.2025, 13:52, "Marcin Maj" ***@***.***>:
How?



pt., 7 mar 2025, 04:02 użytkownik Mido Hamed ***@***.***>

napisał:
